src/main/scala/com/github/unchama/seichiassist/ManagedWorld.scala

```diff
@@ -30,14 +30,17 @@ case object ManagedWorld extends Enum[ManagedWorld] {
   case object WORLD_DOT extends ManagedWorld("world_dot", "ドット絵ワールド")
 
   implicit class ManagedWorldOps(val managedWorld: ManagedWorld) extends AnyVal {
+    /**
+     * 整地ワールドであるかどうか
+     * 整地スキルが使えるかどうか、整地量をカウントするワールドであるかどうかと等しい
```

Comment:
```suggestion
     * これらのワールドは整地スキルが利用可能であり、整地量をカウントするワールドである。
```

私はこちらのほうがより明確だと思います。等しいと書くと読んだ人がミスリーディングを起こす可能性が否定できないためです。